setup.sh

```diff
@@ -190,8 +187,20 @@ main() {
 	printf '\n%s\n\n' "${version} successfully installed, but not yet running"
 	printf '%s\n' "Start the software manually with:"
 	printf '%s\n' "   cd ${SCRIPT_PREFIX}; ./cake-autorate.sh"
-	printf '%s\n' "Run as a service with:"
-	printf '%s\n\n' "   service cake-autorate enable; service cake-autorate start"
+
+	case "$MY_OS" in
```

Comment:
I'm not sure if we setup shellcheck to care, but this should be "${MY_OS}" so stay consistent with the rest of the style.